src/program/types/ObjectExpression.js

```diff
@@ -175,7 +175,15 @@ export default class ObjectExpression extends Node {
 						while (code.original[c] !== ']') c += 1;
 						c += 1;
 					}
-					if (prop.shorthand) {
+					if (prop.key.type === 'Literal' && !prop.computed) {
+						console.log('' + code)
+						code.overwrite(
+							prop.start,
+							prop.key.end + 1,
+							'[' + code.slice(prop.start, prop.key.end) + '] = '
```

Comment:
I think it would be better to remove the trailing whitespace, since hypothetical whitespace between the colon and the property value would be preserved, anyway.